sources-experimental/IABPGroup.cpp

```diff
@@ -148,7 +148,7 @@ IABPGroupRemove::Perform(BMessage*){
 		result = remove->Go(); //sync..
 
 		if(result == 0) {
-			BPAlert* wait = new BPAlert("Remove a group", B_TRANSLATE("\nRemoving..."),NULL,NULL,NULL,B_WIDTH_AS_USUAL, ResourceVectorToBitmap("emblem-delete", 32));
+			BPAlert* wait = new BPAlert("Remove a group", B_TRANSLATE("\nRemoving"B_UTF8_ELLIPSIS),NULL,NULL,NULL,B_WIDTH_AS_USUAL, ResourceVectorToBitmap("emblem-delete", 32));
```

Comment:
I think this will not compile with GCC7 (or GCC2 - some version of GCC has a problem with this).

Reply:
It is strange because PecoRename has the same string + B_UTF8_ELLIPSIS and there are not problem: 
https://github.com/HaikuArchives/PecoRename/blob/686dfb5cb089e061477eb53c652ea73bbd1848cd/Main/MainWindow.cpp#L54
 Do you know the best solution to this problem? string.ReplaceFirst?

Reply:
Isn't it just a warning if you forgot the space in front  of the  B_UTF8_ELLIPSIS ?

Reply:
Is it possible that someone has put the B_UTF8_ELLIPSIS outside the B_TRANSLATE?
Without space the only problem is a warning in gcc7 as point out by @humdingerb.

Reply:
Yes, I meant a warning. I just remembered that some time ago things like this were being fixed in Haiku, but did not have any source at hand.

Reply:
The problem is fixed in 64d32197e014c4c269121d25fada2394cc31e01b. Thanks for the suggestion.